.github/workflows/ci.yml

```diff
@@ -1,50 +1,42 @@
-# This workflow will do a clean install of node dependencies, build the source code and run tests across different versions of node
-# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+name: Continuous integration
 
-name: Node.js CI
-
-on:
-    push:
-        branches: [master]
-    pull_request:
-        branches: [master]
+on: 
+  - push
 
 jobs:
-    build:
-        runs-on: ubuntu-latest
-
-        strategy:
-            matrix:
-                node-version: [12.x, 14.x, 16.x]
-                cucumber-version: [7.x]
```

Comment:
I know we do not support multiple cucumber version since the introduction of cucumber 7 broke the support of the previous ones, but IMO we should still keep this (and other associated steps) since we'll want to support 7 and 8 in the futur.

Reply:
Done
See previous comment about renaming and tidying up